hw/ip/otbn/doc/_index.md

```diff
@@ -372,14 +372,38 @@ The number of entries in the loop stack is implementation-dependent.
 
 ## Design Details
 
-<div class="bd-callout bd-callout-warning">
-  <h5>Note</h5>
+### Error Handling and Reporting
 
-  To be filled in as we create the implementation.
-</div>
+By design, OTBN is a simple processor and provides no error handling support to code that runs on it.
+Any errors will cause OTBN to stop executing and report the failure to the host processor.
+
+Whenever OTBN observes an error, it will generate an alert.
+This gets sent to the alert manager.
+The alert will either be fatal or recoverable, depending on the class of error: see [Alerts]({{< relref "hw/ip/otbn/doc#alerts" >}}) and [Error Conditions]({{< relref "hw/ip/otbn/doc#error-conditions" >}}) below for details.
+
+If OTBN was running when the alert occurred (this is true whenever {{< regref "STATUS.busy" >}} is high), it will also:
+- Immediately stop fetching and executing instructions.
```

Comment:
I did have a half-sentence about avoiding side-effects in there before, let's try to make this as strong as possible.

@GregAC What can we guarantee? Only stop executing instructions? Suppress all/some writebacks?

Reply:
Yeah, I wasn't sure how strong this could be. For example, if we have a shadow register mismatch, I'm not sure how we say whether writebacks were disabled (for which instruction?!). I'll stick in a TODO note after the list to remind us to write something more explicit when we've worked out what we can actually do.

Reply:
Currently we simply bring the state machine to a halt, I don't think error is factored into any enables (so RF, ISPS, Dmem writes can all occur). This reminds me I better open an issue about that!

Reply:
Tracking at #4507